Alloy/commands/compile/ast/controller.js

```diff
@@ -1,8 +1,8 @@
 var U = require('../../../utils'),
-	babylon = require('babylon'),
-	types = require('babel-types'),
-	generate = require('babel-generator').default,
-	traverse = require('babel-traverse').default;
+	babylon = require('@babel/parser'),
+	types = require('@babel/types'),
+	generate = require('@babel/generator').default,
+	traverse = require('@babel/traverse').default;
```

Comment:
@ewanharris - After this line `const { Hub, NodePath } = traverse;` I am getting `undefined` for `Hub`.  Is this related to the `.default`?

Reply:
Yes, `Hub` is a separate export: `const Hub = require('@babel/traverse').Hub;` or `import { Hub } from '@babel/traverse';`

Reply:
Ah!  The error doesn't happen until merged w/ latest master.   After you merge, you can change line to something like this and it works then: 
```
const { Hub, NodePath } = require('@babel/traverse');
```